File: ios/RCTMGL/RCTMGLNativeUserLocationManager.m
#import "RCTMGLNativeUserLocationManager.h"
#import "RCTMGLNativeUserLocation.h"

@implementation RCTMGLNativeUserLocationManager

RCT_EXPORT_MODULE(RCTMGLNativeUserLocation)
RCT_EXPORT_VIEW_PROPERTY(iosShowsUserHeadingIndicator, BOOL)


#pragma - View Properties


#pragma Methods

- (BOOL)requiresMainQueueSetup
{
    return YES;
}

- (UIView *)view
{
    return [[RCTMGLNativeUserLocation alloc] init];
}

@end

File: ios/RCTMGL-v10/RCTMGLNativeUserLocationManager.swift
@objc(RCTMGLNativeUserLocationManager)
class RCTMGLNativeUserLocationManager : RCTViewManager {
  @objc
  override static func requiresMainQueueSetup() -> Bool {
    return true
  }

  @objc override func view() -> UIView {
    return RCTMGLNativeUserLocation()
  }
}
